sql/src/main/java/io/crate/analyze/UnboundAnalyzer.java

```diff
@@ -256,6 +264,30 @@ public AnalyzedStatement visitAlterUser(AlterUser<?> node, Analysis context) {
             return userAnalyzer.analyze((AlterUser<Expression>) node, context.paramTypeHints(), context.transactionContext());
         }
 
+        @Override
+        public AnalyzedStatement visitGrantPrivilege(GrantPrivilege node, Analysis context) {
```

Comment:
Not sure whether it makes sense to make `Privileges` classes generic. I'd keep as it is.